data/yaml/venues.yaml

```diff
@@ -416,8 +419,8 @@ jeptalnrecital:
   oldstyle_letter: F
 knlp:
   acronym: knlp
-  name: 'Knowledgeable NLP: Workshop on Integrating Structured Knowledge
-    and Neural Networks for NLP'
+  name: 'Knowledgeable NLP: Workshop on Integrating Structured Knowledge and Neural
+    Networks for NLP'
```

Comment:
It seems that every time we run black, it switches this. @xinru1414 can you add this to your list of things to look into?

Reply:
Yes will do